docs/embeddings.md

```diff
@@ -265,13 +342,22 @@ palm_embedding = embedding_functions.GooglePalmEmbeddingFunction(
 
 ```
 
-## HuggingFace
```

Comment:
I know you removed this because it doesn't apply or work with the JS client yet, but it looks weird in docs because the side-nav says there is HuggingFace anchor, but when I click on it it leads nowhere.

Can we just update the JS section that HuggingFace inference API is not yet supported by the JS client or perhaps that support for it is coming soon?